tests/components/androidtv/test_media_player.py

```diff
@@ -1050,3 +1058,100 @@ async def test_get_image(hass, hass_ws_client):
     assert msg["success"]
     assert msg["result"]["content_type"] == "image/png"
     assert msg["result"]["content"] == base64.b64encode(b"image").decode("utf-8")
+
+
+async def _test_service(
+    hass,
+    entity_id,
+    ha_service_name,
+    androidtv_method,
+    additional_service_data=None,
+    return_value=None,
+):
+    """Test generic Android TV media player entity service."""
+    service_data = {ATTR_ENTITY_ID: entity_id}
+    if additional_service_data:
+        service_data.update(additional_service_data)
+
+    androidtv_patch = (
+        "androidtv.androidtv_async.AndroidTVAsync"
+        if "android" in entity_id
+        else "firetv.firetv_async.FireTVAsync"
+    )
+    with patch(
+        f"androidtv.{androidtv_patch}.{androidtv_method}", return_value=return_value
+    ) as service_call:
+        await hass.services.async_call(
+            DOMAIN, ha_service_name, service_data=service_data, blocking=True,
+        )
+        assert service_call.called
+
+        # if args or kwargs:
+        #     assert service_call.call_args == call(*args, **kwargs)
+
+
+async def test_services_androidtv(hass):
+    """Test media player services for an Android TV device."""
+    patch_key, entity_id = _setup(CONFIG_ANDROIDTV_ADB_SERVER)
+
+    with patchers.PATCH_ADB_DEVICE_TCP, patchers.patch_connect(True)[patch_key]:
+        with patchers.patch_shell("")[patch_key]:
```

Comment:
It would be nice if the call to `patch_shell` passed a constant to show intent.

Maybe:
```suggestion
        with patchers.patch_shell(SHELL_EMPTY_RESPONSE)[patch_key]:
```

Reply:
I used `SHELL_RESPONSE_OFF` and `SHELL_RESPONSE_STANDBY` in https://github.com/home-assistant/core/commit/2fcfded6c237a4dbb6d53a8c054f0fbd6957cb33